homeassistant/components/ambiclimate/climate.py

```diff
@@ -184,6 +182,10 @@ async def async_update(self) -> None:
             await self._store.async_save(token_info)
 
         data = await self._heater.update_device()
+        if self.min_temp is None:
+            self._attr_min_temp = self._heater.get_min_temp()
+        if self.max_temp is None:
+            self._attr_max_temp = self._heater.get_max_temp()
```

Comment:
Can't these change?

Reply:
I do not think they can change.
But since it does not say anything in the documentation, it might be best to assume they can change